### ISSUE TITLE
Is there any pretrained model for network training?

### ISSUE THREAD

**Issue by yzldw333** [wywu/LAB#5]
I have some doubts about LAB training:
1. Do you use any pretrained models for training. When I saw the prototxt file, I found the risidual unit in hourglass estimator network seens changed(inputchannel->outputchannel/4->outputchannel/4->outputchannel), different with original hourglass network for pose estimation. 

2.Can I train two network seperately, from the paper, I noticed that dfake op is the only reason that the first estimator's training needs the next regressor's output.

3.Finally,  do you have some advice for size reduce of the estimator network. I wrote codes on tf, and I felt it cost long time to load the network structure and do initialization, training process is also slow.

Best regards.
Thank u for your help.

**Comment by wywu** [comment]
Hi,

Thanks for your interest!
1. We trained from scratch and no pre-trained model was used. We modified the vanilla version of stack hourglass to obtain much higher performance.
2. Yes. However, in our experiments, we find that train the boundary estimator and landmark regressor together end-to-end, and then finetune with GAN can obtain the highest accuracy.
3. We have tried to use less channel or use less stacks, however, both of these methods bring some performance degradation. We suggest using less channel rather than cutting stacks.

Best,
Wayne
